scripts/create_baseline_stubs.py

```diff
@@ -79,9 +127,19 @@ def create_metadata(stub_dir: str, version: str) -> None:
     version = match.group(0)
     if os.path.exists(filename):
         return
+    metadata = f'version = "{version}.*"'
+    upstream_repo_url = asyncio.run(get_upstream_repo_url(project))
+    if upstream_repo_url is None:
+        warning = (
+            f"\nDid not succeed in finding a URL pointing to the upstream repo for {project!r}.\n"
```

Comment:
Slightly simpler:

```suggestion
            f"\nDid not succeed in finding a URL pointing to the source code repo for {project!r}.\n"
```

Reply:
Even simpler:
```suggestion
            f"\nDid not succeed in finding the source code repository of {project!r}.\n"
```

Reply:
```suggestion
            f"\nDid not succeed in finding a URL pointing to the source code for {project!r}.\n"
```

Reply:
What do you think of the "Even simpler" suggestion?

Reply:
Nevermind, I think it is clear enough as is.

Reply:
Oh, I didn't see that one! I think I failed to refresh my browser.

Reply:
I pushed 00d2aa5, a slight variant